dune/gdt/playground/spaces/mapper/restricted.hh

```diff
@@ -159,45 +159,40 @@ public:
 
   size_t numDofs(const EntityType& entity) const
   {
-    const auto error_message = check_entity(entity);
-    if (error_message.size() > 0)
-      DUNE_THROW(restricted_space_error, error_message);
+    check_entity(entity);
     return backend().numDofs(entity);
   }
 
   using BaseType::globalIndices;
 
   void globalIndices(const EntityType& entity, DynamicVector<size_t>& ret) const
   {
-    const auto error_message = check_entity(entity);
-    if (error_message.size() > 0)
-      DUNE_THROW(restricted_space_error, error_message);
+    check_entity(entity);
     backend().globalIndices(entity, ret);
     for (size_t ii = 0; ii < numDofs(entity); ++ii)
       ret[ii] = map_to_restricted_.at(ret[ii]);
   }
 
   size_t mapToGlobal(const EntityType& entity, const size_t& localIndex) const
   {
-    const auto error_message = check_entity(entity);
-    if (error_message.size() > 0)
-      DUNE_THROW(restricted_space_error, error_message);
+    check_entity(entity);
     return map_to_restricted_.at(backend().mapToGlobal(entity, localIndex));
   }
 
 private:
-  std::string check_entity(const EntityType& entity) const
+  void check_entity(const EntityType& entity) const
```

Comment:
This was on purpose. If we throw inside `check_entity`, this function name and line number is printed on failuere, as opposed to `numDofs`, `globalIndices`, ... which would give a bit more information. Bu I guess it does not matter much...

Reply:
That's really not worth the overhead if you can just stop the debugger where the exception is thrown though.